cpp/pmtiles.hpp

```diff
@@ -165,11 +198,13 @@ struct entryv3 {
 	}
 };
 
+#ifdef PMTILES_NEED_ENTRYV3_CMP
```

Comment:
Maybe we should just remove this struct if not all consumers need it ( the only significant open-source consumer I'm aware of is https://github.com/felt/tippecanoe/blob/main/pmtiles_file.cpp )

Reply:
hum, for now, I'm only in read-support, but later I'll implement write support as well, and I'll probably need to sort entries too, so it might be useful. So I'm going to remove this #ifdef guard